.gitpod.yml

```diff
@@ -2,7 +2,32 @@ image:
   file: .gitpod.Dockerfile
 
 tasks:
-  - init: |
+  - name: Futurenet
+    init: |
+      docker pull stellar/quickstart:soroban-dev
+    command: |
+      docker run --rm -it \
+      --name stellar \
+      --platform linux/amd64 \
+      -p 8000:8000 \
+      stellar/quickstart:soroban-dev \
+      --futurenet \
+      --enable-soroban-rpc
+  # This "CLI - Futurenet" task opens a terminal for you to interact with the
+  # futurenet network.
+  - name: CLI - Futurenet
+    # We specify some required environment variables for use on the futurenet.
+    env:
+      # This might should be set by the user.
```

Comment:
nit: might or should?

Reply:
oops this was a copy and paste from another file I will fix it.

Reply:
I reworded line 21